src/Commands/utilities/commands.js

```diff
@@ -46,24 +51,25 @@ module.exports = {
 			economyArray = [];
 			otherArray = [];
 			setupArray = [];
+			
 			const promiseArray = client.commands.map(async (c) => {

```

Comment:
Since there is no async code anymore, you could possibly remove the `Promise.all`